src/clr-angular/wizard/_wizard.clarity.scss

```diff
@@ -178,6 +178,15 @@
         transition: box-shadow 0.2s ease-in;
       }
 
+      &.error {
```

Comment:
This implementation (does not match the final design)[https://github.com/vmware/clarity/issues/2203#issuecomment-408176349]. It needs to be corrected.

Reply:
My implementation was based on this comment: https://github.com/vmware/clarity/issues/2203#issuecomment-481379574 and the design given in the issue description. Should I respect the other one?

Reply:
I think you misread the comment. The comment you linked to states that the design with the icon next to "Very Beautiful Images" is the one that needs to be implemented. The design in the description up top doesn't have that text in the stepnav.

Reply:
The wrong design is still being implemented. Please [review the finalized design and correct this](https://github.com/vmware/clarity/issues/2203#issuecomment-408176349).

Thanks!

Reply:
You are right! I misread the comment. 
This is how the feature looks now: 
![Screenshot from 2019-09-30 18-10-33](https://user-images.githubusercontent.com/53469024/65893090-0fc79c80-e3b0-11e9-994c-7a6fcf8d8ce4.png)
What should I change in order to respect the final design?

Reply:
That's the correct design. Thanks for doing that, @florin-borceanu .